### ISSUE TITLE
GT  - Error on rendering in Latex with addition of rowname_col

### ISSUE THREAD

**Issue by polc1410** [rstudio/gt#841]
When rendering a gt table in latex - the data is incorrectly structured - placing the wrong information in the wrong table

Basic data
```
require(tidyverse)
require(gt)
tibble (
    hospital = c("A", "B", "C"),
    `2019` = c(100, 150, 200),
    `2021` = c(110, 160, 210)
    ) %>%
    mutate(change = (`2021`-`2019`)/`2019`) -> myTibble
```

This table - sort of works:
```
myTibble %>%
    gt()%>% #rowname_col = "hospital") %>%
   cols_label(hospital = "Hospital", change = "Change (increase)") %>%
    cols_align(align = "left", columns = c('hospital')) %>%
    cols_align(align = "right", columns = c('change')) %>%
    fmt_percent(columns = c(change), decimals = 1)%>%
    tab_header(title = "Overall change in activity", subtitle= "Jan - Aug 2019 vs Jan - Aug 2021") %>%
    grand_summary_rows(
    columns = c('2019', '2021'), 
    fns = list(Total = ~sum(., na.rm=T)), formatter = fmt_number, decimals = 0
  ) 
```

but the grand summary doesn't work?

If we add a rownname_col to the gt, the layout in Rstudio is good. In the rendered PDF - is plain wrong:

```
myTibble %>%
    gt(rowname_col = "hospital") %>%
   cols_label(hospital = "Hospital", change = "Change (increase)") %>%
    cols_align(align = "left", columns = c('hospital')) %>%
    cols_align(align = "right", columns = c('change')) %>%
    fmt_percent(columns = c(change), decimals = 1)%>%
    tab_header(title = "Overall change in activity", subtitle= "Jan - Aug 2019 vs Jan - Aug 2021") %>%
    grand_summary_rows(
    columns = c('2019', '2021'), 
    fns = list(Total = ~sum(., na.rm=T)), formatter = fmt_number, decimals = 0
  ) 
 ```   

It should actually render like this

![Rplot](https://user-images.githubusercontent.com/1819213/137649510-5ef3ed84-dca3-4aca-9769-c652e152d7a2.png)

[ErrorReport.pdf](https://github.com/rstudio/gt/files/7361077/ErrorReport.pdf)
:

**Comment by rich-iannone** [comment]
We will have a closer look at this. Thanks for reporting!

**Comment by rich-iannone** [comment]
@polc1410 We did some work recently to fix (and seriously improve!) LaTeX table output and so this code now produces the expected results. Here's a screenshot of a PDF render I made using your code:

<img width="979" alt="summaries_latex_gt" src="https://user-images.githubusercontent.com/5612024/151575890-c627ffda-88d4-46c5-af10-a1d563c0d5c9.png">

To get this working, install the dev version of gt with `devtools::install_github()`. These improvements will eventually be in the v0.4.0 CRAN release.
